unitary/alpha/quantum_world.py

```diff
@@ -11,13 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Dict, List, Optional, Sequence, Union
```

Comment:
Out of curiosity, why did you move these lower?

Reply:
I just used pylint, to check the code. For some reason it likes if the standard libraries are imported first. 
So, what I wanted to do is moving "import cirq" lower, but it seems I copy-pasted another line, too :)